Lesson34/selcan/lesson34-app/src/components/Form/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Account } from "../Account";
+
+// TODO: Create a new Component that contains log in form, so that user can type username and password and click submit button to log in.
+// If user is logged in, show Account component.
+// If user is logged out, show Log In component.
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [loggedIn, setLoggedIn] = useState("");
```

Comment:
Add dispatch context to use it later to update user information in the store:
```
const dispatch = useContext(UserDispatchContext);
```
Make sure to import useContext and UserDispatchContext;